tasks/create_ocp_infra_nodes.yml

```diff
@@ -0,0 +1,121 @@
+---
+- name: import shift_stack_vars data
+  include_vars:
+    dir: "{{ playbook_dir }}/vars"
+    files_matching: shift_stack_vars.yaml
+
+- name: Get cluster name
+  shell: |
+    {%raw%}oc get machineset -n openshift-machine-api -o=go-template='{{ (index (index .items 0).metadata.labels {%endraw%} "{{ machineset_metadata_label_prefix }}/cluster-api-cluster" {%raw%}) }}'{%endraw%}
```

Comment:
why are using {%raw%}?

Reply:
> why are using {%raw%}?

This task comes from original playbook[1]
[1] https://github.com/redhat-performance/jetpack/blob/master/create_ocp_infra_nodes.yml#L13